### DISCUSSION TITLE
Donation Rewards - new options.

### DISCUSSION THREAD

**Discussion by Vyperhand** [chromiecraft/chromiecraft#5492] [Suggestions]
Things I would happily donate for a chance to acquire:

-Reputation Tokens for core factions.   As long as these are core-faction only (like Darkspear Trolls, Undercity, Ironforge, etc.)  this should avoid a pay-to-win stigma, as the primary benefit would be to use another race's mount.

-Alternatively, make sure that there is at least one mount sample of each race's mount available in the shop.  Right now only a few races are represented in the selections.

**Comment by BeefSteaker** [comment]
Home city reps can be maxxed in a few hours by doing starter zone quests. 

There should the unarmoured swift mounts for every race except blood elf and draenei that don't have them

**Reply by Vyperhand** [reply]
Almost - there are no human/undead mounts available in the donation shop.  Looks like all other classic race options are covered.  Regardless, thank you very much for the response.

**Comment by Nyeriah** [comment]
Thanks for your suggestion but I’m afraid power ups like reputation tokens and other things that affect the game play directly are not things we’ll be seeing in the reward catalogue as we’ve a strictly non-profit, non p2w philosophy

**Reply by Vyperhand** [reply]
Thank you for the response, I can understand.  There may be some benefit to race rep other than mount access that I'm not remembering.
